nautobot/core/templates/inc/dynamic_groups_panel.html

```diff
@@ -1,15 +0,0 @@
-{% load helpers %}
```

Comment:
Might need to leave this file, just in case we have plugin templates that are expecting to use it?